files/qsub.pl

```diff
@@ -190,7 +189,8 @@ sub make_command
         'p=i'      => \$priority,
         'pe=s{2}'  => \@pe_ev_opts,
         'P=s'      => \$wckey,
-        'q=s'      => \$destination,
+        'q=s'      => sub { fatal("In SLURM no queues defined. " .
```

Comment:
"should be "no queues are defined."

Also, didn't we plan to make this a warning, and simply ignore whatever is passed to -q`?

Reply:
See my comment at the directives. I prefer to have uniform behavior.

Reply:
Very much +1 on uniform behavior, I overlooked that the `#PBS` lines in the job script are handled elsewhere... Early exiting with an error is definitely better if we can't control that easily.